packages/@aws-cdk/aws-codebuild/README.md

```diff
@@ -280,11 +280,12 @@ can use the `environment` property to customize the build environment:
 ## Images
 
 The CodeBuild library supports both Linux and Windows images via the
-`LinuxBuildImage` and `WindowsBuildImage` classes, respectively.
+`LinuxBuildImage`, `LinuxArmBuildImage`, and `WindowsBuildImage` classes, respectively.
```

Comment:
Let's use parenthesis here instead, otherwise this sentence reads weirdly:

```suggestion
`LinuxBuildImage` (or `LinuxArmBuildImage`), and `WindowsBuildImage` classes, respectively.
```